operator-wg/whitepaper/Operator-WhitePaper_v1-0.md

```diff
@@ -109,9 +107,9 @@ The Operator pattern consists of three components:
 * The application or infrastructure that we want to manage.
 * A domain specific language that enables the user to specify the desired state of the application in a declarative way.
 * A controller that runs continuously:
-  * Reads and is aware of the state.
-  * Runs actions against the application in an automated way.
-  * Report the state of the application  in a declarative way.
+    * Reads and is aware of the state.
+    * Runs actions against the application in an automated way.
```

Comment:
It's not always "against the application". It could be any operations, like cleaning up S3 buckets.
Maybe rephrase like: "Runs actions against change of operations state in an automated way".

Reply:
Suggestion applied https://github.com/cncf/tag-app-delivery/pull/114/commits/713d692a5c3851de7e28173c9f26b655eb5619f0 - Thanks!